sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/SkipLocalsInitAttribute.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The original file has https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/SkipLocalsInitAttribute.cs#L1 is it a problem if we change it? I'd say not because in order to shim they are expected to be copied.

Reply:
Given SkipLocalsInit becomes only relevant for slightly larger stack alloc byte arrays we might also decide to get rid of it depending on the discussion (or consider it slightly too esoteric)

Reply:
I'm totally ok with that